pallets/elections/Cargo.toml

```diff
@@ -37,8 +38,12 @@ std = [
     "pallet-session/std",
     "pallet-staking/std",
     "pallet-authorship/std",
+    "pallets-support/std",
```

Comment:
Could you please bump `version`? Tbh not sure whether this is breaking change or not, but I guess `0.4.0` would to the job.

Reply:
sure, done